test/recipes/30-test_evp_data/evpciph_des3_common.txt

```diff
@@ -22,6 +22,8 @@ Ciphertext = 3FE301C962AC01D02213763C1CBD4CDC799657C064ECF5D41C673812CFDE9675
 NextIV = 1c673812cfde9675
 
 # DES EDE3 ECB test
+# FIPS 3.0.0 has a bug in the IV length #17591
+FIPSversion = >3.0.0
```

Comment:
I have a feeling the next test also failed?

Reply:
It passes here :)
Easy to address if it does fail after CIs are introduced.